src/main/java/software/sigma/internship/entity/Locale.java

```diff
@@ -0,0 +1,32 @@
+package software.sigma.internship.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity(name = "locale")
+@Table(name = "locale")
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+public class Locale {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @Column(name = "iso_code", nullable = false, length = 2)
+    private String isoCode;
+
+    @OneToMany(mappedBy = "locale")
+    private List<User> users;
```

Comment:
We don't need a user list in the locale entity. Moreover, be cautious with ToString annotation as it will fall with StackOverflowException